WordPress/Classes/Utility/ReachabilityUtils+OnlineActions.swift

```diff
@@ -43,15 +43,14 @@ extension ReachabilityUtils {
     ///
     /// We use a Snackbar instead of a literal Alert because, for internet connection errors,
     /// Alerts can be disruptive.
-    static func showNoInternetConnectionNotice() {
-        let notice = Notice(title: NoConnectionMessage.title,
-                            message: NoConnectionMessage.message,
-                            tag: NoConnectionMessage.tag)
+    static func showNoInternetConnectionNotice(message: String) {
```

Comment:
Im curious, why did you update this method to take a message when we never really call it wit ha custom message?
Won't this method be called with the default message?

Reply:
This method is used by `NetworkAwareUI.presentNoNetworkAlert()` where it passes `noConnectionMessage()`:

```swift
    func presentNoNetworkAlert() {
        ReachabilityUtils.showNoInternetConnectionNotice(message: noConnectionMessage())
    }
```

`NetworkAwareUI` is designed to allow overrides of `noConnectionMessage()`  like what we do here in `PageListViewController`:

```swift
    override func noConnectionMessage() -> String {
        return NSLocalizedString("No internet connection. Some pages may be unavailable while offline.",
                                 comment: "Error message shown when the user is browsing Site Pages without an internet connection.")
    }
```

This is what changes the _default_ offline messages. 

I'm not very happy with how this looks. There's a lot of misdirection. I'm not quite sure we can make this better.

Reply:
Ok gotcha. Yeah lots of  misdirections :/